thirdparty/build-thirdparty.sh

```diff
@@ -502,6 +502,7 @@ build_brpc() {
     rm -rf CMakeCache.txt CMakeFiles/
     LDFLAGS="-L${TP_LIB_DIR} -static-libstdc++ -static-libgcc" \
     ${CMAKE_CMD} -G "${GENERATOR}" -DBUILD_SHARED_LIBS=0 -DCMAKE_INSTALL_PREFIX=$TP_INSTALL_DIR \
+    -DCMAKE_LIBRARY_PATH="$TP_INSTALL_DIR/lib;$TP_INSTALL_DIR/lib64" \
```

Comment:
```suggestion
    -DCMAKE_LIBRARY_PATH="$TP_INSTALL_DIR/lib64" \
```

`lib` is just a soft link of `lib64`